rc_django/cache_implementation/memcache.py

```diff
@@ -18,22 +18,23 @@
 
 
 class MemcachedCache(object):
+    def __init__(self):
+        self.client = self._get_client()
```

Comment:
Would this change make all the method calls run with the same thread unless one creates a MemcachedCache object for each access?